content/en/docs/pipelines/sdk/pipelines-metrics.md

```diff
@@ -21,30 +21,42 @@ pipeline agent uploads the local file as your run-time metrics. You can view the
 uploaded metrics as a visualization in the **Runs** page for a particular
 experiment in the Kubeflow Pipelines UI.
  
-## Export the metrics file
+## Export the metrics dictionary
 
-To enable metrics, your program must write out a file named 
-`/mlpipeline-metrics.json`. For example:
+To enable metrics, your component must have an output called "MLPipeline Metrics".
+JSON-serialized metrics dictionary should be returned from that output.
 
 ```Python
-  accuracy = accuracy_score(df['target'], df['predicted'])
+from kfp.components import InputPath, OutputPath, create_component_from_func
+
+def produce_metrics(mlpipeline_metrics_path: OutputPath('Metrics')):
+  import json
+  
+  accuracy = 0.9
   metrics = {
     'metrics': [{
       'name': 'accuracy-score', # The name of the metric. Visualized as the column name in the runs table.
       'numberValue':  accuracy, # The value of the metric. Must be a numeric value.
       'format': "PERCENTAGE",   # The optional format of the metric. Supported values are "RAW" (displayed in raw format) and "PERCENTAGE" (displayed in percentage format).
     }]
   }
-  with file_io.FileIO('/mlpipeline-metrics.json', 'w') as f:
+  with open(mlpipeline_metrics_path, 'w') as f:
     json.dump(metrics, f)
+
+produce_metrics_op = create_component_from_func(
+    produce_metrics,
+    base_image='python:3.7',
+    packages_to_install=[],
+    output_component_file='component.yaml',
+)
 ```
 
 See the 
 [full example](https://github.com/kubeflow/pipelines/blob/master/components/local/confusion_matrix/src/confusion_matrix.py).
```

Comment:
Suggestion:

- Add an explanation of the example, so that: "...[full example]... of a program that generates confusion matrix data from prediction results."
- For accessibility, swap "See the..." with "Refer to the..."
```suggestion
Refer to the 
[full example](https://github.com/kubeflow/pipelines/blob/master/components/local/confusion_matrix/src/confusion_matrix.py)
of a program that generates confusion matrix data from prediction results.
```

Reply:
Done